assets/src/styles/pages/_site-location.scss

```diff
@@ -18,3 +18,30 @@
     height: auto;
   }
 }
+
+#banner-title {
+    float: right;
+    font-family: calibri,tahoma,arial,verdana,sans-serif;
+    font-size: 1.7em;
+    text-decoration: none;
+
+    a {
+        color: #233f8f !important;
+    }
+    .ngwmn-highlight {
+        color: #d1544a;
+    }
+}
+@media only screen 
+    and (min-width:701px)
+    and (max-width:800px) {
+    #banner-title {
+        font-size: 23px;
+    }
+}
+@media only screen 
+    and (max-width:700px) {
+    #banner-title {
+        font-size: 18px;
+    }
+}
```

Comment:
I think this should be scoped inside `.banner-container` in _header.scss rather than `_site-location.scss`. This one is scoped specifically to the site location page, here: https://github.com/ACWI-SOGW/ngwmn-ui/blob/master/assets/src/styles/main.scss#L35-L37 Which would mean that the styles won't apply on the provider pages.